src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -0,0 +1,121 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import PIDControl.PIDControl;
+import PIDControl.PIDControl.Coefficient;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Arm extends SubsystemBase 
+{
+    private static Arm _instance;
+
+    public static Arm getInstance()
+    {
+        if(_instance == null) 
+        {
+            _instance = new Arm();
+        }
+
+        return _instance;
+    }
+
+    //Extension Motor
+    private DigitalInput     _limitSwitch;
+    private CANSparkMax      _linearMotor;
+    private RelativeEncoder  _extensionEncoder;
+    private PIDControl       _extensionPid;
+    private boolean          _extensionPidActive;
+
+    //Shoulder Motor
+    private DutyCycleEncoder _pitchEncoder;
+    private CANSparkMax      _pitchMotor;
+    private CANSparkMax      followerPitchMotor;
+    private PIDControl       _shoulderPid;
+    
+   
+
```

Comment:
We can also clean up a little bit of whitespace here. I think one empty line is sufficient. Three lines is a tad overkill